plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -99,6 +99,28 @@ module.exports = class ServiceTree extends Tree {
     });
   }
 
+  getItemParent(id, parent = null) {
+    if (this.getId() === id) {
+      return parent;
+    }
+
+    if (!this.list || this.list.length === 0) {
+      return null;
+    }
+
+    let parentFound = null;
+
+    this.list.forEach(child => {
+      if (child instanceof ServiceTree) {
```

Comment:
If we have nested groups here, I believe we can run in a bug where I have `parentFound = this` but it is the first element on the list, so I keep searching further, even when I already found the parent. This cause me to return null.

For instance, given this tree, the test `getItemParent("/test/foo")`, fails.
```js
      this.instance = new ServiceTree({
        id: "/",
        items: [
          {
            id: "/test",
            items: [
              {
                id: "/test/foo"
              },
              {
                id: "/test/bar"
              },
              {
                id: "/test/baz/boo",
                items: [
                  {
                    id: "/test/baz/boo/1"
                  }
                ]
              }
            ]
          },
          {
            id: "/alpha",
            cmd: "cmd"
          },
          {
            id: "/beta",
            cmd: "cmd",
            items: {
              DCOS_PACKAGE_FRAMEWORK_NAME: "beta"
            }
          }
        ]
      });
```

a solution would be: 
changing line 114 to: ` if (child instanceof ServiceTree && parentFound == null) {`

I tried changing 114 to return this, but it did not work :/